<!DOCTYPE HTML>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
  <title>COMBIGS | Database and tools:: SASTRA Deemed to be University</title>
  <link rel="icon" href="images/sastra.png" />
  <link rel="stylesheet" href="layout/styles/framework.css">
  <link rel="stylesheet" href="layout/styles/layout.css">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <!-- <script src="path/to/js/disable.js"></script> -->
  <style>
    .hamburger {
      display: none;
    }
    @media screen and (max-width: 768px) {
      #mainav ul {
        display: none;
      }
      .hamburger {
        display: block;
        cursor: pointer;
      }
    }
    .table-img {
      width: 150px;
      height: 50px;
      object-fit: contain; /* This ensures the image maintains its aspect ratio without stretching */
    }
    .database-tools-table td {
      text-align: justify;
      padding: 8px;
    }
  </style>
</head>

<body>
  <div class="wrapper row1">
    <header id="header" class="hoc clear">
      <!-- 
        <div id="logo" class="fl_left">
          <a href="home.html"><img src="COMBIGS_Website_2022\www.sastra.edu\combigs\images\combigs.png" alt=""/></a>
        </div> 
      -->
      <!-- Hamburger Icon -->
      <div class="hamburger fl_right" onclick="toggleNav()">â˜°</div>
      <nav id="mainav" class="fl_right">
        <ul class="clear">
          <li><a href="home.html">Home</a></li>
          <li><a href="magazine.html">Magazine</a></li>
          <li><a href="event.html">Event Archives</a></li>
          <li><a href="experiences.html">Experience</a></li>
          <li><a href="intern.html">Internships</a></li>
          <li class="active"><a href="dbtools.html">Databases and Tools</a></li>
          <li><a href="members.html">Members</a></li>
        </ul>
      </nav>
    </header>
  </div>
  
  <div class="wrapper bgded" style="background-image:url('./images/dbtools-web-banner.png'); background-size:cover; background-position:center center;"> 
    <section id="pageintro" class="hoc clear" style="padding: 60px 0;">
      <div style="background-color: rgba(255, 255, 255, 0.7); display: inline-block; padding: 10px; border-radius: 5px;">
      `<h2 class="heading" style="color: #000; text-shadow: 2px 2px 4px rgba(255,255,255,0.5);">Databases and Tools</h2>
          <p style="color: #000; text-shadow: 2px 2px 4px rgba(255,255,255,0.5);">
            Bridging computational techniques with biological insights to advance scientific discovery.
          </p>
      </div>
    </section>
  </div>  
  
  <div class="wrapper row3">
    <main class="hoc container clear">
      <div class="content">
        <h1>Databases and Access Links</h1>
        <div class="scrollable">
          <table class="database-tools-table">
            <thead>
              <tr>
                <th>Name of the Database</th>
                <th>Description</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td align="center">
                  <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">
                    <img src="images/NCBI.jpg" class="table-img">
                  </a>
                </td>
                <td>
                  <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">NCBI</a> is a source for public biomedical databases and has software tools for analyzing molecular and genomic data.
                </td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="http://www.uniprot.org/" target="_blank">  
                      <img src="images/UNIPROT.png" class="table-img">
                  </a>
                </td>
                <td>The Universal Protein Resource (<a class="l" href="http://www.uniprot.org/" target="_blank">UniProt</a>) is a comprehensive resource for protein sequence and annotation data.</td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="http://trace.ddbj.nig.ac.jp/dra/index_e.html" target="_blank">
                    <img src="images/DDBJ.png" class="table-img">  
                  </a>
                </td>
                <td>
                  The DNA Data Bank of Japan (<a class="l" href="http://trace.ddbj.nig.ac.jp/dra/index_e.html" target="_blank">DDBJ</a>) provides freely available nucleotide sequence data.</td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="https://www.ebi.ac.uk/chembl/" target="_blank">
                    <img src="images/CHEMBL.png" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="http://www.uniprot.org/" target="_blank">ChEMBL</a> is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs.
                </td>
              </tr>
            
              <tr>
                <td align=center> 
                  <a  href="http://www.treefam.org/search" target="_blank">
                    <img src="http://www.treefam.org/static/images/treefam_logo.png" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="http://www.treefam.org/search" target="_blank">TreeFam</a> (Tree families database) is a database of phylogenetic trees of animal genes. It provides information about ortholog and paralog assignments and evolutionary history of gene families.
                </td>
              </tr>
            
              <tr>
                <td align=center>
                  <a  href="https://www.rcsb.org/pdb/home/home.do" target="_blank">
                    <img src="https://cdn.rcsb.org/rcsb-pdb/v2/common/images/rcsb_logo.png" class="table-img">
                  </a>
                </td>
                <td>
                  The Protein Data Bank (<a class="l" href="https://www.rcsb.org/pdb/home/home.do" target="_blank">PDB</a>) is a crystallographic database for the three-dimensional structural data of large biological molecules, such as proteins and nucleic acids. The data, typically obtained by X-ray crystallography, NMR spectroscopy, or, increasingly, cryo-electron microscopy.The PDB is a key resource in areas of structural biology, such as structural genomics. </td>
                </tr>
              <tr>
                <td align=center>
                  <a  href="http://www.cathdb.info/" target="_blank">
                    <img src="images/CATH_-_Protein_Structure_Classification_Database.png" class="table-img">
                  </a>
                </td>
                <td>The <a class="l" href="http://www.cathdb.info/" target="_blank">CATH</a> database provides hierarchical classification of protein domains based on their folding patterns. Domains are obtained from protein structures deposited in the Protein Data Bank and both domain identification and subsequent classification use manual as well as automated procedures.</td>
              </tr>
              <tr>
                <td align=center>  
                  <a  href="http://scop.berkeley.edu/" target="_blank">
                    <img src="https://scop.mrc-lmb.cam.ac.uk/static/media/scop_logo.358e74fa.png" class="table-img">
                  </a>
                </td>
                <td>The Structural Classification of Proteins (<a class="l" href="http://scop.berkeley.edu/" target="_blank">SCOP</a>) database is a largely manual classification of protein structural domains based on similarities of their structures and amino acid sequences. A motivation for this classification is to determine the evolutionary relationship between proteins.The shapes of domains are called "folds" in SCOP. Domains belonging to the same fold have the same major secondary structures in the same arrangement with the same topological connections</td>
              </tr>
              <tr>
                <td align=center>
                  <a  href="https://www.expasy.org/" target="_blank">
                    <img src="https://www.expasy.org/images/expasy-homepage.png" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://www.expasy.org/" target="_blank">Expasy</a> is an integrated database containing information about genomes, proteomes, evolution and structural biology.</td>
              </tr>
              <tr>
                <td align=center>
                  <a  href="https://www.genome.jp/kegg/" target="_blank">
                    <img src="https://www.genome.jp/Fig/kegg128.gif" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="https://www.genome.jp/kegg/" target="_blank">KEGG</a> (Kyoto Encyclopedia of Genes and Genomes) is a collection of databases dealing with genomes, biological pathways, diseases, drugs, and chemical substances.</td>
               </tr>
              <tr>
                <td align=center> 
                  <a  href="https://prosite.expasy.org/" target="_blank">
                    <img src="https://prosite.expasy.org/images/prosite/prosite.gif" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="https://prosite.expasy.org/" target="_blank">PROSITE</a> is a protein database. It consists of entries describing the protein families, domains and functional sites as well as amino acid patterns and profiles in them.</td>
              </tr>
              <tr>
                <td align=center>
                  <a  href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank">
                    <img src="images/GenBank.png" class="table-img">
                  </a>
                </td>
                <td>
                  The <a class="l" href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank">GenBank</a> sequence database is an open access, annotated collection of all publicly available nucleotide sequences and their protein translations. It is a comprehensive source of stable automatic annotation of individual genomes, and of the synteny and orthology relationships between them.
                </td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="https://www.ensembl.org/index.html" target="_blank"><img src="images/ensembl_logo.png" class="table-img"></a></td>
                <td><a class="l" href="https://www.ensembl.org/index.html" target="_blank">Ensembl</a> is a bioinformatics project to organize biological information around the sequences of large genomes.
                </td>
              </tr>
              <tr>
                <td align=center>
                  <a  href="https://www.ebi.ac.uk/interpro/" target="_blank">
                    <img src="images/INTERPRO.jpg" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="https://www.ebi.ac.uk/interpro/" target="_blank">InterPro</a> is a database of protein families, domains and functional sites in which identifiable features found in known proteins can be applied to new protein sequences in order to functionally characterize them.
                </td>
              </tr>
              <tr>
                <td align=center>
                  <a href="https://string-db.org/" target="_blank"><img src="images/STRING&#32;DATABASE.png" class="table-img"></a></td>
                <td>
                  <a class="l" href="https://string-db.org/" target="_blank">STRING</a> is a database of known and predicted protein-protein interactions.
                </td>
              </tr>
              <tr>
                <td align=center>
                  <a href="http://www.bioinfo.pte.hu/more/TrEMBL.htm" target="_blank">
                    <img src="images/TrEMBL.png" style="width: 100px; height: 50px;">
                  </a>
                </td>
                <td>
                  <a class="l" href="http://www.bioinfo.pte.hu/more/TrEMBL.htm" target="_blank">TrEMBL</a> is a computer-annotated protein sequence database supplementing the SWISS-PROT Protein Sequence Data Bank.</td>
              </tr>
            </tbody>
          </table>
        </div>

        <br>

        <h1>Tools and Access Links</h1>
        <div class="scrollable">
          <table class="database-tools-table">
            <thead>
              <tr>
                <th>Name of the Tool</th>
                <th>Description</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td align="center">
                  <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">
                    <img src="images/blast-logo.jpg" class="table-img">
                  </a>
                </td>
                <td>
                  <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a> finds regions of similarity between biological sequences.
                </td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="http://www.genome.jp/tools-bin/clustalw" target="_blank">
                    <img src="images/clustalo_title.png" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="http://www.genome.jp/tools-bin/clustalw" target="_blank">Clustal</a> is a series of widely used computer programs used in Bioinformatics for multiple sequence alignment.In many cases, the input set of query sequences are assumed to have an evolutionary relationship by which they share a lineage and are descended from a common ancestor. From the resulting MSA, sequence homology can be inferred and phylogenetic analysis can be conducted to assess the sequences' shared evolutionary origins.</td>
              </tr>
              <tr>
                <td align=center> 
                  <a  href="https://swissmodel.expasy.org/" target="_blank">
                    <img src="https://swissmodel.expasy.org/static/images/sib_logo_small.svg" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="https://swissmodel.expasy.org/" target="_blank">SWISS-MODEL</a> is a structural bioinformatics web-server dedicated to homology modeling of protein 3D structures.
                </td>
              </tr>
              <tr>
                <td align=center>
                  <a href="https://saves.mbi.ucla.edu/" target="_blank">
                    <img src="images/saves.png" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://saves.mbi.ucla.edu/" target="_blank">SAVES</a> tools consist of many protein varification tools. The input structure should be in pdb format.</td>
              </tr>
              
              <tr>
                <td align=center>
                  <a  href="https://www.ebi.ac.uk/Tools/emboss/" target="_blank">
                    <img src="images/EMBOSS&#32;DATABASE.jpg" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://www.ebi.ac.uk/Tools/emboss/" target="_blank">EMBOSS</a> is the European Molecular Biology Open Software Suite. EMBOSS programs contains collection of tools for sequence analysis.</td>
              </tr>
            
              <tr>
                <td align=center>
                  <a class="l" href="https://www.ebi.ac.uk/Tools/msa/muscle/" target="_blank">
                    <img src="images/muscle.png" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://www.ebi.ac.uk/Tools/msa/muscle/" target="_blank">MUSCLE</a> stands for MUltiple Sequence Comparison by Log Expectation. It is used for multiple sequence alignment of protein and nucleotide sequences.</td>
              </tr>
              <tr>
                <td align=center>
                  <a class="l" href="https://www.compbio.dundee.ac.uk/jpred/" target="_blank">
                    <img src="https://www.compbio.dundee.ac.uk/jpred/myimages/Jpred4_logo8_3.svg" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://www.compbio.dundee.ac.uk/jpred/" target="_blank">JPred</a> is a Protein Secondary Structure Prediction server and has been in operation since approximately 1998.</td>
              </tr>
             
              <tr>
                <td align=center>  
                  <a href="https://proteininformationresource.org/" target="_blank">
                    <img src="https://proteininformationresource.org/pirwww/images/temp/PIRBanner.png" class="table-img">
                  </a>
                </td>
                <td>The Protein Information Resource (<a class="l" href="https://proteininformationresource.org/" target="_blank">PIR</a>) is an integrated public resource of protein informatics that supports genomic and proteomic research and scientific discovery.</td>
              </tr>
            
              <tr>
                <td align=center>
                  <a class="l" href="https://pymol.org/2/#download" target="_blank">
                    <img src="https://upload.wikimedia.org/wikipedia/commons/8/87/PyMOL_logo.svg" class="table-img">
                  </a>
                </td>
                <td>
                  <a class="l" href="https://pymol.org/2/#download" target="_blank">PyMOL</a> is a molecular visualization system used to view and ivestigate atoms and molecules.
                </td>
              </tr>
              <tr>
                <td align=center> 
                  <a class="l" href="https://avogadro.cc/" target="_blank">
                    <img src="https://upload.wikimedia.org/wikipedia/commons/c/c1/Avogadro.png" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://avogadro.cc/" target="_blank">Avagadro</a> is a molecular editor sued in computational chemistry, molecular modelling to inverstigate and alter molecules and atoms.</td>
              </tr>
              <tr>
                <td align=center>
                  <a href="https://usegalaxy.org/" target="_blank">
                    <img src="images/GALAXY.jpg" class="table-img">
                  </a>
                </td>
                <td><a class="l" href="https://usegalaxy.org/" target="_blank">GALAXY</a> is a next generation Scientific workflow, data integration and data analysis publishing platform which aims to make computational biology accessible to research Scientists</td>
              </tr>
              <tr>
                <td align=center>
                  <a href="https://www.gromacs.org/" target="_blank">
                  <img src="https://www.gromacs.org/_static/gmx_logo_blue.png" class="table-img">
                </a>
              </td>
                <td><a class="l" href="https://www.gromacs.org/" target="_blank">Gromacs</a> is a molecular dynamic package designed for simulations of protein, lipids and nucleic acid. </td>
              </tr>
              <tr>
                <td align=center> 
                  <a class="l" href="https://bernhardcl.github.io/coot/wincoot-download.html" target="_blank">
                    <img src="images/WINCOOT.png" class="table-img">
                  </a>
                </td>
                <td>Crystallographic Object-Orinted Toolkit(<a class="l" href="https://bernhardcl.github.io/coot/wincoot-download.html" target="_blank">COOT</a>) used to display and manipulate atomic models of macromolecules like proteins and nucleic acids using 3D computer graphics. </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
      <div class="clear"></div>
    </main>
  </div>
  
  
  <div class="wrapper row5">
    <footer id="footer" class="hoc clear">
      <p class="fl_left">COMBIGS, SASTRA Deemed to be University, All rights Reserved. 2023</p>
      <p class="fl_right">Developed by <a target="_blank" href="https://www.sastra.edu/">SASTRA Web Team</a></p>
    </footer>
  </div>
  
  <a id="backtotop" href="#top"><i class="fas fa-chevron-up"></i></a>

  <script>
    function toggleNav() {
      var nav = document.getElementById("mainav").getElementsByTagName('ul')[0];
      if (nav.style.display === "block") {
        nav.style.display = "none";
      } else {
        nav.style.display = "block";
      }
    }
  
    // Remove inline styles on window resize
    window.addEventListener('resize', function() {
      var nav = document.getElementById("mainav").getElementsByTagName('ul')[0];
      if (window.innerWidth > 768) {
        nav.removeAttribute('style');
      }
    });
  </script>
  
</body>
</html>
